integrations/oci/pom.xml

```diff
@@ -41,5 +41,6 @@
         <module>vault-health</module>
         <module>cdi</module>
         <module>atp</module>
+        <module>metrics</module>
```

Comment:
README.md needed? Or any other form of documentation do we want to loop into this change?

Reply:
Will be added in the helidon doc in the near future. I will create a doc issue for it.